dobson/utils.py

```diff
@@ -0,0 +1,62 @@
+import configparser
+import json
+import os
+import subprocess
+from collections import namedtuple
+
+
+Device = namedtuple('Device', ['presence', 'user', 'model'])
+
+config = configparser.ConfigParser()
+config.read('dobson/config.ini')
+
+
+SLACK_API_TOKEN = config['dobson']['SlackApiToken']
+
+# Only allow querying for presence in certain channels
+ALLOWED_CHANNEL_IDS = json.loads(config['dobson']['AllowedChannelIds'])
```

Comment:
I feel like doing something like [this](https://stackoverflow.com/a/11296626/1979001) where it just loads and splits by newline would be a bit better here (but I'm open to suggestions on that of course). Then it doesn't need to be an array in the config file, and also isn't prone to some problem where it interprets it as some other JSON data structure, which could be hard to debug since it'll still load ok but just fail in some later step.

Reply:
Fair enough, that's equally readable and doesn't introduce the json dependency, so I'll modify it to that format.